src/common/entity/binary_sensor_icon.ts

```diff
@@ -16,9 +16,13 @@ export const binarySensorIcon = (state: HassEntity) => {
     case "garage_door":
       return activated ? "hass:garage" : "hass:garage-open";
     case "gas":
+      return activated ? "hass:shield-check" : "hass:biohazard";
     case "power":
+      return activated ? "hass:power-off" : "hass:power";
     case "problem":
+      return activated ? "hass:check-circle" : "hass:alert-circle";
     case "safety":
+      return activated ? "hass:check-circle" : "hass:alert-circle";
```

Comment:
The icon for smoke looks more fitting for safety?

Reply:
Id like that for Gas...

maybe simply use

```
return activated ? "hass:shield-check" : "hass:shield-alert";
```

for safety...